modules/ROOT/pages/zuora/zuora-connector-studio.adoc

```diff
@@ -0,0 +1,50 @@
+
+= Zuora Studio Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+To configure a connector in Anypoint Studio, follow this process:
+
+. Install the connector.
+. Configure the connector.
+. Configure an input source for the connector.
+
+== Install the Connector
+
+. In Studio, create a Mule Project.
+. Click the Exchange *(X)* icon in the Studio task bar.
+. In Exchange, click *Login* and supply your Anypoint Platform username and password.
+. In Exchange, select *All assets* and search for *Zuora Connector*.
+. Select *Zuora Connector*, and click *Add to project*.
+. Follow the prompts to install the connector.
+
+== Configure the Connector
+
+To configure this connector:
+
+. Drag and drop a connector operation into the Studio canvas.
+. To create a global element for the connector, set these fields:
++
+** *basePath*: Base path of the REST API for your instance. Usually this is the `/rest` path.
+** *username*: Zuora username.
+** *password*: Zuora password.
+** *host*: Path to the Zuora API, for example: `apisandbox-api.zuora.com`.
+** *port*: Port where the Zuora API listens for requests.
+** *protocol*: Protocol to use for accessing the Zuora API.
++
+
+image::zuora/zuora-basic-auth-studio.png[Zuora Connector Configuration in Studio 7]
```

Comment:
Again, just not so sure how helpful this image is here. I was told by George to remove all of them in the FD and Studio templates.